experiments/AMIP/modular/Project.toml

```diff
@@ -50,44 +50,27 @@ UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 ArgParse = "1.1"
 ArtifactWrappers = "0.2"
 AtmosphericProfilesLibrary = "0.1"
-ClimaAtmos = "0.15.1"
 CLIMAParameters = "0.4, 0.5, 0.6, 0.7"
-ClimaComms = "0.3, 0.4"
-ClimaCore = "0.10.40"
+ClimaComms = "0.4, 0.5"
```

Comment:
This could be removed if we update the top-level Project

Reply:
Yeah, sorry, it must have slipped. I removed it now